src/util/sss_iobuf.c

```diff
@@ -19,18 +19,18 @@ struct sss_iobuf {
 };
 
 struct sss_iobuf *sss_iobuf_init_empty(TALLOC_CTX *mem_ctx,
-                                       size_t size,
-                                       size_t capacity)
+                                     size_t size,
+                                     size_t capacity)
```

Comment:
why indentation change?

Reply:
Arghhh! I changed the my IDE's font and now I see this aligned even if it not. Fixing this and changing the IDE's font.